documentation/docs/tirith-policies/tirith-policy-conditions.md

```diff
@@ -35,4 +35,25 @@ import TabItem from '@theme/TabItem';
       "item": "https://www.lambdatest.com/support/docs/tirith-policy-conditions/"
     }]
   })
-}}></script>
\ No newline at end of file
+}}></script>
+Conditions define how values from input JSON are evaluated. Tirith supports various condition types:
+
+- **Equals :** Checks if a value matches a specified value.
+- **RegexMatch :** Validates if a string matches a regex pattern.
+- **LessThan / GreaterThan :** Compares numeric values.
+
+```javascript title="policy.json"
+{
```

Comment:
could you place a ... in the top and the bottom to signify that this is an incomplete policy?